modules/reserved-ips/variables.tf

```diff
@@ -0,0 +1,126 @@
+##############################################################################
+# VPC Variables
+##############################################################################
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
+  type        = string
+  default     = "vpe"
+}
+
+variable "vpc_name" {
+  description = "Name of the VPC where the Endpoint Gateways will be created. This value is used to dynamically generate VPE names."
```

Comment:
Again, we need to make it clear this is not used if the `vpe_name` option in the `cloud_services` input variable is used